src/ngraph/runtime/cpu/CMakeLists.txt

```diff
@@ -195,7 +200,10 @@ if (NGRAPH_CPU_ENABLE)
     endif()
 
     add_dependencies(cpu_backend ext_mkldnn ext_eigen)
-    target_link_libraries(cpu_backend PUBLIC ngraph libmkldnn libeigen libjson libtbb)
+    target_link_libraries(cpu_backend PUBLIC ngraph libmkldnn libeigen libjson)
+    if (NGRAPH_TBB_ENABLE)
+        target_link_libraries(cpu_backend PUBLIC libtbb)
```

Comment:
I think this should be PRIVATE. Probably should make libmkldnn, libeigen and libjson PRIVATE too. codegen too.

Reply:
Had to make it PUBLIC again as it breaks compiling a unit test.